src/collections/Form/Form.js

```diff
@@ -7,8 +8,22 @@ export default class Form extends Component {
   static propTypes = {
     children: PropTypes.node,
     className: PropTypes.string,
+    settings: PropTypes.object,
   };
 
+  componentDidMount() {
+    this.element = $(this.refs.element);
+    this.element.form(this.props.settings);
+  }
+
+  componentWillUnmount() {
+    this.element.off();
+  }
+
+  plugin() {
+    return this.element.form(...arguments);
+  }
+
```

Comment:
Pretty self explanatory, on mount we init the Semantic UI form plugin with the settings passed in.  We clean it up on unmount.  This allows us to support all Semantic UI form plugin settings and framework consistencies.